synapse/storage/schema/main/delta/68/02_delete_account_data_for_deactivated_accounts.sql

```diff
@@ -0,0 +1,20 @@
+/* Copyright 2021 The Matrix.org Foundation C.I.C
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+-- We want to retroactively delete account data for users that were already
+-- deactivated.
+INSERT INTO background_updates (ordering, update_name, progress_json) VALUES
+  (6520, 'delete_account_data_for_deactivated_users', '{}');
```

Comment:
```suggestion
  (6802, 'delete_account_data_for_deactivated_users', '{}');
```
Should this be 6802, to match the schema version and file name?

Reply:
Good spot, forgot to update that, thanks!
I've also bumped this to `03` since someone took `02`, just for the sake of it.